tests/fixtures/basic/archive/test/law-html/hooks/fsmonitor-watchman.sample

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/perl
```

Comment:
I imagine that none of these Git hooks are ever used? I wonder if they could just be deleted? Or ignored?